bin/check-emr-cluster.rb

```diff
@@ -73,20 +74,6 @@ class CheckEMRCluster < Sensu::Plugin::Check::CLI
          default: -1,
          proc: proc(&:to_i)
 
-  option :warning_under,
```

Comment:
Why the removal of these 2 options?

Reply:
The underlying functionality was removed.